react-carousel-component/my-app/src/Carousel.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import PrevButton from './PrevButton';
+import NextButton from './NextButton';
+import Content from './Content';
+import Indicators from './Indicators';
+import './Carousel.css';
+
+export default function Carousel({ images }) {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(handleClickNext, 3000);
+    return () => clearInterval(interval);
+  });
+
+  function handleClickPrev() {
+    const index = (currentIndex - 1 + images.length) % images.length;
+    setCurrentIndex(index);
+  }
+
+  function handleClickNext() {
```

Comment:
You should find, when you fix the dependency in useEffect, that this now causes the effect to run on every render. Do you remember how to fix that?